packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/grafana/dashboards-1.14/alertmanager-overview.yaml.patch

```diff
@@ -13,11 +13,603 @@
    name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" $) "alertmanager-overview" | trunc 63 | trimSuffix "-" }}
    annotations:
  {{ toYaml .Values.grafana.sidecar.dashboards.annotations | indent 4 }}
-@@ -611,4 +612,5 @@
-         "uid": "alertmanager-overview",
-         "version": 0
-     }
```

Comment:
Should we keep this indentifier?

Reply:
Had to remind myself of what thats used for: https://grafana.com/docs/grafana/latest/developers/http_api/dashboard/#identifier-id-vs-unique-identifier-uid. Correct me if I'm wrong, but looks like we do want to keep the `uid` set in case there is another deployment. Even if we don't necessarily support that, we probably would like to prevent that from hurting our functionality too much.